sdk/evaluation/azure-ai-evaluation/azure/ai/evaluation/_evaluators/_eci/_eci.py

```diff
@@ -7,7 +7,7 @@
 from azure.ai.evaluation._evaluators._common import RaiServiceEvaluatorBase
 
 
-class ECIEvaluator(RaiServiceEvaluatorBase):
+class ECIEvaluator(RaiServiceEvaluatorBase[str]):
```

Comment:
not experienmental?

Reply:
since it's private, we probably don't need to mark it experimental. Only for public-facing things

Reply:
marked it anyways